test/e2e_main_test.ts

```diff
@@ -24,8 +23,7 @@ describe('toClosureJS', () => {
     if (!closure) {
       diagnostics.forEach(v => console.log(JSON.stringify(v)));
       assert.fail();
-      // TODO(lucassloan): remove when the .d.ts has the correct types
-      return {compiledJS: '', sourceMap: new SourceMapConsumer('' as any)};
+      return;
```

Comment:
The compiler caught it. My life goal is to replace myself with machine automation.  ;)